samples/rust does not build

~/mystikos/samples/rust$ make
rustc ./hello_world.rs
make: rustc: Command not found
Makefile:5: recipe for target 'all' failed
make: *** [all] Error 127

--- a/samples/rust/Makefile
+++ b/samples/rust/Makefile
@@ -1,6 +1,10 @@
 TOP = $(abspath ../..)
 include $(TOP)/defs.mak
 
+ifeq (, $(shell which rustc))
+ $(error "Please install rustc first!")
+endif
+
 all:
 	rustc ./hello_world.rs
 

